docs/modules/layout.md

```diff
@@ -0,0 +1,20 @@
+## Project layout
+### Module layout
+    modules                    # The folder containing the code for modules in this suite
+        mongodb                # driver
+        mongodb_storage        # key-value
+        mongodb_watchdog       # logger
+    .coveralls.yml             # Code coverage configuration file
+    .scrutinizer.yml           # Scrutinizer configuration file
+    .travis.yml                # Travis build configuration  file
+    README.md                  # Current Readme file
+    README.old.md              # Old readme (for developers' reference)
+    composer.json              # Composer configuration file
+    core.phpunit.xml           # PHPUnit configuration file
+    example.settings.local.php # Example settings to connect to MongoDB
+
+### Documentation layout
+    mkdocs.yml    # The configuration file.
```

Comment:
Yes. We need to remove code from mkdocs branch. But, shouldn't we do this after we are done with this ticket and documentation updates? Since I am not sure if we delete it now and when merging it with 8.x-2.x branch would GIT delete the code in  8.x-2.x too?

Reply:
Sure, after the merge.